src/lwc/utilTemplateBuilder/utilTemplateBuilder.js

```diff
@@ -130,6 +132,7 @@ const EXCLUDED_BATCH_HEADER_FIELDS = [
     DI_BATCH_DEFAULTS_INFO.fieldApiName,
     DI_BATCH_GIFT_ENTRY_VERSION_INFO.fieldApiName,
     DI_BATCH_FORM_TEMPLATE_INFO.fieldApiName,
+    DI_BATCH_TABLE_COLUMNS_FIELD.fieldApiName,
```

Comment:
Could we use this opportunity to include Created Date, Created By ID, Last Modified Date, and Last Modified by ID in the excluded fields here? These are generally restricted due to being read-only but some users see them appear when the have enabled the ability to edit system audit fields on their profile

Reply:
Added a collection of system fields below!